.github/workflows/backport-maintenance.yml

```
@@ -0,0 +1,150 @@
+env:
+  # Name of the maintenance branch we want to backport to.
+  TARGET_BRANCH: v0.5.0
+
+jobs:
+  backport:
+    # Run only when the PR was merged in main and labeled for backporting.
+    # Accepted labels: backport-<TARGET_BRANCH> or dependencies.
+    if: |
+      github.event.pull_request.merged == true &&
+      github.event.pull_request.base.ref == 'main' &&
+      (
```

actionlint (1.7.12)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)